src/plots/cartesian/dragbox.js

```diff
@@ -434,13 +434,6 @@ function makeDragBox(gd, plotinfo, x, y, w, h, ns, ew) {
 
         recomputeAxisLists();
 
-        // if the plot has scrollbars (more than a tiny excess)
-        // disable scrollzoom too.
-        if(pc.scrollHeight - pc.clientHeight > 10 ||
```

Comment:
Hmm. But the comment says if the **plot* had scrollbars, not if the page has some.

Maybe we aren't using the correct DOM properties.

Reply:
So in https://codepen.io/plotly/pen/QjKZbg, 

on full-screen I got:

![image](https://user-images.githubusercontent.com/6675409/47802513-cc2f6300-dd06-11e8-95df-0f1eaf007e79.png)

but after resize to half-screen-width, I get e.g.

![image](https://user-images.githubusercontent.com/6675409/47802562-e8330480-dd06-11e8-9e6a-5eefb502b663.png)

----

So perhaps we should compare `pc.scrollWidth` with `fullLayout.width` and `pc.scrollHeight` with `fullLayout.height` to determine if scroll should be activated.